# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLQueueParamTraits.h
# Commit: 6149f5b15c5e
# Full Hash: 6149f5b15c5e278ef15ea36fa896fa8c146ebdca
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2021-01-14 08:32:45
# Regressor Bug: 1640607
# File Overlap Count: 1
# Description:
#   Bug 1640607 - Send SurfaceDescriptors for GPU blitting for video-to-webgl. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101061
# ==============================================================================

diff -r 29fd3a6a7e27 -r 6149f5b15c5e dom/canvas/WebGLQueueParamTraits.h
--- a/dom/canvas/WebGLQueueParamTraits.h	Wed Jan 13 20:14:54 2021 +0000
+++ b/dom/canvas/WebGLQueueParamTraits.h	Wed Jan 13 21:49:40 2021 +0000
@@ -68,6 +68,8 @@
 };
 template <>
 struct IsTriviallySerializable<webgl::ReadPixelsDesc> : std::true_type {};
+template <>
+struct IsTriviallySerializable<layers::SurfaceDescriptor> : std::true_type {};
 
 template <typename T>
 struct QueueParamTraits<RawBuffer<T>> {